jhipster-jdl.jdl

```diff
@@ -76,34 +76,93 @@ entity CompanionDiagnosticDevice {
     manufacturer String
 }
 
-entity PremarketApproval {
-	device String
-    genericName String
-    applicant TextBlob
+enum FdaSubmissionType {
+	PMA, DE_NOVO, HDE, FIVE_TEN_K
```

Comment:
@calvinlu3 is it appropriate to name it FIVE_TEN_K ?

Reply:
The FDA also uses `PMN` (Premarket Notification).
https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm?ID=K173492

Reply:
Yeah I saw that too. But the abbreviation seems only be used in the URL?